webassembly-language-support.md

```diff
@@ -22,6 +22,8 @@ For each environment, we use the following icons to indicate a level of support:
 
 The Fermyon Platform requires [WASI](https://wasi.dev) support. Any language that has a ✅ for WASI should be supported on the Fermyon Platform.
 
+>> If you are interested in contributing to this guide, head on over to [the GitHub repo](https://github.com/fermyon/wasm-languages).
```

Comment:
Can/should we also update the link here: https://github.com/fermyon/wasm-languages/blob/main/webassembly-language-support.md#updates-and-additions ?

See https://github.com/fermyon/wasm-languages/issues/11

Reply:
@vdice good idea. I updated this with a more substantial block of text point here to this project, to Discord, and to Twitter.

Closes #11